build/helper/documentation_snippets.py

```diff
@@ -96,5 +96,135 @@
 { 'simulate': True, 'driver_setup': { 'Model': '<model number>',  'BoardType': '<type>' } }
 '''
 
+default_initiate_function_doc = '''
+Calls initiate
+'''
+
+initiate_function_note = '''
+This function will return a Python context manager that will initiate on entering and abort on exit.
+'''
+
+def initiate_function_def_for_doc(functions, config):
+    # This is very specific to IVI. We look for a 'close' function and if we find it,
```

Comment:
`to IVI` -> `to the IVI state model`, which not all drivers in nimi-python follow.